README.md

```diff
@@ -24,4 +24,5 @@ Please follow the established naming conventions present in the code, it makes i
 - [x] Make linux binaries/executables with next release
 - [x] Add ability for program to save all calculations done to a file (python read/write)
 - [x] Refactor if-else-elif statements to use new "match" statement introduced in Python 3.10
+- [ ] Add a safety feature(not just intentionally crashing users) for "match" statements when user is not using Python 3.10 or above(add a legacy component that still uses if-else-elif statements could be a good route)
```

Comment:
```suggestion
- [ ] Add a safety feature (not just intentionally crashing users) for "match" statements when user is not using Python 3.10 or above (add a legacy component that still uses if-else-elif statements could be a good route)
```
This would conform it to the parentheses style used in earlier TODOs.